controllers/tunnelbinding_controller.go

```diff
@@ -588,6 +587,7 @@ func (r *TunnelBindingReconciler) configureCloudflareDaemon() error {
 			finalIngresses = append(finalIngresses, UnvalidatedIngressRule{
 				Hostname:      binding.Status.Services[i].Hostname,
 				Service:       targetService,
+				Path:          subject.Spec.Path,
```

Comment:
The `omitempty` is only defined on the CR, but passed concretely to the `UnvalidatedIngressRule` which would currently get the default value for `Path` which is "".

Reply:
Maybe I'm missing what you are trying to say,  but `yaml.Marshall` w/ `omitempty` will omit key when value is `""` entirely when marshalling, which is what it'll be right?

Reply:
Oh right, the `UnvalidatedIngressRule` also has `omitempty`, sorry. Looks good then, will test it out over the weekend, thanks!

Reply:
thx! running a local build myself already